resources/rafter/charts/rafter-asyncapi-service/README.md

```diff
@@ -0,0 +1,10 @@
+# Rafter AsyncAPI Service
```

Comment:
I would suggest changing the chart name from `charts/rafter-asyncapi-service` to `charts/asyncapi-service`. What follows, please update the README title to `# AsyncAPI Service.`

Reply:
Changed